<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deviceServerClass</class>
 <widget class="QMainWindow" name="deviceServerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>deviceServer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>396</width>
     <height>423</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="Por">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Port">
        <property name="text">
         <string>6000</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ServerListen">
        <property name="text">
         <string>启动监听</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>客户端01:</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTableWidget" name="raiseWgt"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>客户端02:</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTableWidget" name="raiseWgt_2"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>总耗时</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="totalTime">
        <property name="styleSheet">
         <string notr="true">
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>平均耗时</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="avgTime">
        <property name="styleSheet">
         <string notr="true">
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>成功率</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="successRate">
        <property name="styleSheet">
         <string notr="true">
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>栏杆机:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="up">
        <property name="text">
         <string>抬杆</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="down">
        <property name="text">
         <string>落杆</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stressTest">
        <property name="text">
         <string>压力测试</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="makeReport">
        <property name="text">
         <string>生成报表</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>车检状态:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="heartStatus">
        <property name="text">
         <string>暂停心跳</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="comboBox"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="vehCount"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="client"/>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="status"/>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>车检:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>车检数量:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>客户端连接数：:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>396</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
